pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -280,7 +280,7 @@ func (n *NetworkPolicyController) createAppliedToGroup(np *networkingv1.NetworkP
 		UID:      types.UID(appliedToGroupUID),
 		Selector: *groupSelector,
 	}
-	klog.V(2).Infof("Creating new AppliedToGroup %s with selector (%s)", newAppliedToGroup.Name, newAppliedToGroup.Selector.NormalizedName)
+	klog.Infof("Creating new AppliedToGroup %s with selector (%s)", newAppliedToGroup.Name, newAppliedToGroup.Selector.NormalizedName)
```

Comment:
Maybe these should stay at their current verbosity level, since they are about internal objects that the user is not likely to be familiar with / interested in?

Reply:
Makes sense, done.